source/system-administrators/activities/reindexing-content-in-prod.rst

```diff
@@ -4,95 +4,223 @@
 Reindexing Content Without Disrupting Service in Production
 ===========================================================
 
-In some scenarios it's not possible/appropriate to delete a live index and wait for the index to rebuild in production.  Perhaps the index is
-driving dynamic features on the site that will break while the index is empty or being rebuilt.  In these scenarios you need a process for building
-the index off line and swapping it in.
+In some scenarios it's not possible/appropriate to delete a live index and wait for the index to rebuild in production. 
+Perhaps the index is driving dynamic features on the site that will break while the index is empty or being rebuilt.  
+In these scenarios you need a process for building the index off line and swapping it in.
+
+-------------------
+Using Elasticsearch
+-------------------
+
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+Step 1: Create a new index
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The first step is to create a new index on Elasticsearch where you can index the content:
+
+#. Check the next version for the index name, you can find the current name using the following command:
+   
+   .. code-block:: bash
+      :linenos:
+   
+      curl -s http://ES_HOST:ES_PORT/_cat/aliases | grep SITE_NAME
+     
+      # Example result
+     
+      SITE_NAME   SITE_NAME_v1   - - -
+
+#. Find the appropriate settings for the environment and Crafter CMS version being updated, for example the latest
+   version can be found `here <https://github.com/craftercms/search/tree/develop/crafter-search-elasticsearch/src/main/resources/crafter/elasticsearch>`_
+#. Create the new index using the next version and the settings:
+   
+   .. code-block:: bash
+      :linenos:
+   
+      curl -s -X PUT http://ES_HOST:ES_PORT/SITE_NAME_v2 -H 'Content-Type: application/json' -d '{ JSON_SETTINGS }'
+
+.. note::
+  You can also download the required settings file and use the reference to the file: ``curl ... -d '@filename'``
```

Comment:
I think it's just better to tell people from the start to download the file and send the curl with the file, instead of putting a note.